controllers/feeds/getFeedsV2.js

```diff
@@ -195,7 +195,7 @@ module.exports = async (req, res) => {
             newItem = await modifyAnonimityPost(newItem, isBlurredPost);
             newItem = modifyReactionsPost(newItem, newItem.anonimity);
             if (item.verb === POST_VERB_POLL) {
-              const postPoll = await modifyPollPostObject(req.userId, item);
+              const postPoll = await modifyPollPostObject(req.userId, item, isBlurredPost);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `modifyPollPostObject` function now takes an additional argument `isBlurredPost`. Ensure that all other usages of this function are updated to include this new argument to maintain consistency and avoid potential errors.
```diff
- const postPoll = await modifyPollPostObject(req.userId, item);
+ const postPoll = await modifyPollPostObject(req.userId, item, isBlurredPost);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->